.dockerignore

```diff
@@ -0,0 +1,16 @@
+/assets/.cache
```

Comment:
Is there not a way to make the `.dockerignore` copy `.gitignore`? Keeping the two in sync will be annoying.

Reply:
They do two different things, they don't necessarily need to be the same. Dockerignore prevents files from being used during build, which speeds up the build and makes smaller layers. But honestly, if we always build the official image from a clean repository, we don't even need a dockerignore file.